backend/makeMigrate.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
```

Comment:
I would not use this shebang but rather `#!/usr/bin/env python3`.

Reply:
this does not work on my ubuntu/Pycharm config